lib/shoulda/matchers/active_record/have_db_column_matcher.rb

```diff
@@ -238,6 +240,19 @@ def correct_primary?
           end
         end
 
+        def correct_array?
+          return true unless @options.key?(:array)
+
+          if matched_column.array.to_s == @options[:array].to_s
+            true
+          else
+            @missing = "#{model_class} has a db column named #{@column} " <<
+                       "of array #{matched_column.array}, " <<
+                       "not #{@options[:array]}."
```

Comment:
Use 2 (not 11) spaces for indenting an expression in an assignment spanning multiple lines.